app/docs/0.10.x/proxy.md

```diff
@@ -598,7 +599,31 @@ More information on this topic is covered in the
 
 [Back to TOC](#table-of-contents)
 
-#### 4. Response
+#### 4. Errors & retries
+
+Whenever an error occurs during while proxying, Kong will use the underlying
+Nginx [retries][ngx-http-proxy-retries] to pass the request onto the next
+upstream.
+
+There are two configurable elements here:
+
+1. The number of retries. This can be configured per API using the `retries`
```

Comment:
"The number of retries." is a sentance fragment. consider refactoring this to be a complete sentence